Plugin/clFindResultsStyler.cpp

```diff
@@ -155,12 +158,12 @@ void clFindResultsStyler::StyleText(wxStyledTextCtrl* ctrl, wxStyledTextEvent& e
                 ctrl->SetStyling(1, LEX_FIF_DEFAULT);
             } else {
                 // File name
-                filenameStyleLen = 1;
+                filenameStyleLen = chWidth;
                 m_curstate = kFile;
             }
             break;
         case kLineNumber:
-            ++lineNumberStyleLen;
+            lineNumberStyleLen += chWidth;
```

Comment:
why do you increase the line number by the char width here?

Reply:
That's just to make it consistent look with other changes. The outcome should be same though (chWidth == 1), but I'll revert it back if it doesn't make sense.

Reply:
Since we are advancing line numbers it makes no sense to make it depends on the char width (even though it will always be 1). For clarity, it should be `++lineNumberStyleLen`

Reply:
Okay, reverted two of them.